crates/valence_new/examples/text.rs

```diff
@@ -50,10 +50,10 @@ fn init_clients(
 
         client.send_message("Welcome to the text example.".bold());
         client
-            .send_message("The following examples show ways to use the different text components.");
+            .send_message(Text::from("The following examples show ways to use the different text components."));
 
         // Text examples
-        client.send_message("\nText");
+        client.send_message(Text::from("\nText"));
```

Comment:
You should make `From` impls so this works without changing all these lines.

Reply:
Ok

Reply:
It would be not possible to implement From<&str> for Cow<Text>

Reply:
Hmm you're right. I still don't want it to be necessary to wrap messages in `Text::from`, that severely degrades dev UX. Maybe you could keep `send_message(text: Into<Text>)` and just make it a `Cow` internally? It might be better to make a wrapper type for `Cow<'a, Text>`. @rj00a thoughts?

Reply:
The `Text` API will probably need a thorough redesign at some point. Maybe we could make a dedicated trait like `IntoText` to hide the details between `Text`, `&Text`, `Cow<Text>`, etc. For now, let's just keep `impl Into<Text>` and only change the packets.